src/sas/qtgui/Utilities/WhatsNew/WhatsNew.py

```diff
@@ -135,6 +136,7 @@ def __init__(self, parent=None, strictly_newer=True):
 
         # other buttons
         self.buttonLayout.addSpacerItem(QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum))
+        self.buttonLayout.addWidget(self.prevButton)
         self.buttonLayout.addWidget(self.nextButton)
 
```

Comment:
This layout is becomng more complex with each release. Maybe it's time to spin off the UI into a separate file?
Adding widgets and layouts manualy is haphazard.
Just a suggestion